src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/LanguageServices/VisualBasicSemanticFactsService.vb

```diff
@@ -50,6 +52,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Protected Overrides Function ToIdentifierToken(identifier As String) As SyntaxToken
+            Return identifier.ToIdentifierToken
```

Comment:
```suggestion
            Return identifier.ToIdentifierToken()
```

Reply:
I'll do it as part of a follow-up change.

Reply:
absolutely.  for nits, please just do later.  i know how long it takes to rerun CI, so i don't want to slow you down :)